locale/en/submission.po

```diff
@@ -2430,3 +2452,5 @@ msgstr "Competing Interests"
 msgid "author.competingInterests.description"
 msgstr "Please disclose any competing interests this author may have with the research subject."
 
+msgid "submission.localeNotSupported"
+msgstr "The language of the submission ({$language}) is not one of the supported submission languages. You can still edit the submission details but are not currently accepting new submissions with this language."
```

Comment:
There's a missing word here at `but [missing word] are not currently accepting`.